tests/testthat/test_isSimpleNumeric.R

```diff
@@ -0,0 +1,22 @@
+context("isSimpleNumeric")
+
+test_that("isSimpleNumeric", {
+  ps = makeParamSet()
+  expect_true(isSimpleNumeric(ps))
+  
+  ps = makeParamSet(
+    makeNumericParam("x", lower=1, upper=2)
+  )
+  expect_true(isSimpleNumeric(ps))
+  
+  ps = makeParamSet(
+    makeDiscreteParam("x", values=1:2)
+  )
+  expect_false(isSimpleNumeric(ps))
+  
+  ps = makeParamSet(
+    makeDiscreteParam("x", values=1:2),
```

Comment:
nitpicking: this could be `makeNumericParam("x", lower=1, upper =2)` to emphasize what we are testing and expecting.